--- sbpp_discord.sp	2026-06-21 01:41:20.933186739 +0200
+++ sbpp_discord.sp.fixed	2026-06-21 01:50:04.944216622 +0200
@@ -282,7 +282,7 @@
 		LogError("HTTP request failed for %s against %s", sAuthor, sTarget);
 }
 
-public int OnHTTPRequestComplete(Handle hRequest, bool bFailure, bool bRequestSuccessful, EHTTPStatusCode eStatusCode, int iClient, int iTarget)
+public void OnHTTPRequestComplete(Handle hRequest, bool bFailure, bool bRequestSuccessful, EHTTPStatusCode eStatusCode, int iClient, int iTarget)
 {
 	if (!bRequestSuccessful || eStatusCode != k_EHTTPStatusCode204NoContent)
 	{
